### ISSUE TITLE
Dictionary not fully supported for certain words

### ISSUE THREAD

**Issue by wenlzhang** [sylvainhalle/textidote#148]
Although the word "optimisation" was included in `dict_UK.txt`, when checking with the following configuration, it still gives the following checking hint:
```
--output html --read-all
--dict dict_UK.txt
--remove IEEEeqnarray,IEEEeqnarray*
--check en_UK example.tex
```
<img width="348" alt="Screenshot 2020-09-28 at 21 35 24" src="https://user-images.githubusercontent.com/44930227/94478016-f0d23500-01d2-11eb-932b-72c310f23a18.png">

**Comment by grisuthedragon** [comment]
We had the same issue that our wordlists are not read. The calling sequence is nearly the same.

**Comment by sylvainhalle** [comment]
I mark this as a duplicate of #142; the fix #143 resolves this issue as well as the other bug.
